src/platform/plugins/shared/workflows_execution_engine/server/templating_engine.ts

```diff
@@ -18,8 +18,8 @@ export class WorkflowTemplatingEngine {
       strictVariables: false,
     });
 
-    // register json filter that converts JSON string to object
-    this.engine.registerFilter('json', (value: unknown): unknown => {
+    // register json_parse filter that converts JSON string to object
+    this.engine.registerFilter('json_parse', (value: unknown): unknown => {
```

Comment:
What do you think if we add unit-tests for Templating engine?
I think it's a good point to start, because Templating engine now is more stable.

Reply:
Added!